src/main/java/keycontacts/storage/JsonAdaptedMakeupLesson.java

```diff
@@ -0,0 +1,84 @@
+package keycontacts.storage;
+
+import java.time.LocalDate;
+
+import keycontacts.commons.exceptions.IllegalValueException;
+import keycontacts.model.lesson.Lesson;
+import keycontacts.model.lesson.MakeupLesson;
+import keycontacts.model.lesson.Time;
+
+/**
+ * Jackson-friendly version of {@link MakeupLesson}.
+ */
+public class JsonAdaptedMakeupLesson {
+    public static final String MISSING_FIELD_MESSAGE_FORMAT = "Makeup lesson's %s field is missing!";
+
+    private final String lessonDate;
+    private final String startTime;
+    private final String endTime;
+
+    /**
+     * Constructs a {@code JsonAdaptedMakeupLesson} with no arguments. This is used by Jackson to create a new
+     * {@code JsonAdaptedMakeupLesson} object.
+     */
+    public JsonAdaptedMakeupLesson() {
+        this.lessonDate = "";
+        this.startTime = "";
+        this.endTime = "";
+    }
```

Comment:
Why is this constructor needed?

Reply:
The tests won't compile if this default constructor is not here

Reply:
This should not be the case. Probably the sign of some underlying error

Reply:
ok, removed!